hard-protocol-home.js

```diff
@@ -255,15 +265,29 @@ const setApyDiplayValues = async (denoms, siteData, cssIds) => {
       const denomPrice = prices[rewardDenom].price;
       numerator += Number(reward[rewardDenom].amount) * denomPrice;
     }
-    const denomenator = balanceCurrency * collatDenomPrice;
+    const denominator = balanceCurrency * collatDenomPrice;
     let apy = '0.00%';
 
-    if (denomenator !==0) {
+    if (denominator !==0) {
       // use usdFormatter to truncate to 2 decimals and round
-      const apyWithDollarSign = usdFormatter.format((numerator / denomenator) * 100);
+      const apyWithDollarSign = usdFormatter.format((numerator / denominator) * 100);
       apy = formatPercentage(noDollarSign(apyWithDollarSign));
     }
-    const cssId = cssIds[denom]['apy'];
+    const cssId = cssIds[denom]['rapy'];
+    setDisplayValueById(cssId, apy)
+  }
+};
+
+const setSupplyApyDisplayValue = async (denoms, siteData, cssIds) => {
+  const interestRates = siteData['supplyApyByDenom'];
```

Comment:
with the changes above, this will change to `const interestRates = siteData['interestRates']`